src/main/java/cat/udl/eps/softarch/mytournamentx/domain/TeamInvitation.java

```diff
@@ -7,38 +7,26 @@
 import javax.validation.constraints.NotBlank;
 import java.util.Date;
 
-@Entity @IdClass(TeamInvitationId.class)
+@Entity
 @Data
 @EqualsAndHashCode(callSuper = true)
-public class TeamInvitation extends UriEntity<TeamInvitationId> {
+public class TeamInvitation extends UriEntity<String> {
 
     @Id
-    @NotBlank
-    private String teamId;
-
-    @Id
-    @NotBlank
-    private String userId;
+    private String id;
```

Comment:
This should be an autogenerated ID. Otherwise, it will be necessary to set this id manually, which is not what we want.